src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -54,225 +63,234 @@ const MapView: React.FC = () => {
+        });
 
-    // Loads the objects into the layer
-    blocksLayer.loadGeoJson(
-      'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/blocks.json',
-    );
+        // Creates a new layer
+        const privateStreetsLayer = new google.maps.Data({ map });
 
-    // Sets the style of the layer to colored blocks with black outline
-    function setBlocksStyle(v: boolean) {
-      blocksLayer.setStyle((feature) => {
-        let color = 'green';
+        // Loads the objects into the layer
+        privateStreetsLayer.loadGeoJson(
+          'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/private_streets.json',
+        );
 
-        // Use this for coloring reserved/completed blocks a different color
-        if (feature.getProperty('ID') % 10 === 0) {
-          color = 'yellow';
+        // Sets the style of the layer to simple red lines
+        function setPrivateStreetsStyle(v: boolean) {
+          privateStreetsLayer.setStyle({
+            strokeColor: 'red',
+            strokeWeight: 2,
+            visible: v,
+          });
         }
 
-        if (feature.getProperty('ID') % 10 === 1) {
-          color = 'red';
+        // Initially false while the neighborhoods are shown
+        setPrivateStreetsStyle(false);
+
+        // Creates a new layer
+        const blocksLayer = new google.maps.Data({ map });
+
+        // Loads the objects into the layer
+        blocksLayer.addGeoJson(blocks);
+
+        // Sets the style of the layer to colored blocks with black outline
+        function setBlocksStyle(v: boolean) {
+          blocksLayer.setStyle((feature) => {
+            let color = 'green';
+
+            // Use this for coloring reserved/completed blocks a different color
+            if (feature.getProperty('ID') % 10 === 0) {
+              color = 'yellow';
+            }
+
+            if (feature.getProperty('ID') % 10 === 1) {
+              color = 'red';
+            }
+
+            // Use this for selecting blocks
+            if (feature.getProperty('PRECINCT') === 'YES') {
+              color = 'black';
+            }
+
+            // Set styling here
+            return {
+              fillColor: color,
+              strokeColor: 'black',
+              strokeWeight: 1,
+              visible: v,
+            };
+          });
         }
 
-        // Use this for selecting blocks
-        if (feature.getProperty('PRECINCT') === 'YES') {
-          color = 'black';
+        // Initially false while the neighborhoods are shown
+        setBlocksStyle(false);
+
+        // Creates a new layer
+        const neighborhoodsLayer = new google.maps.Data({ map });
+
+        // Function: Returns name in shortHand
+        /*
+        function shortHand(name: string): string {
+          for (const shName of shortHandNames) {
+            if (shName.key === name) {
+              return shName.value;
+            }
+          }
+          return name;
+        }
+        */
```

Comment:
Do we need this?

Reply:
The original neighborhood names are pretty wordy so we will want to incorporate this in some way.